classes/plugins/Plugin.php

```diff
@@ -344,25 +344,30 @@ public function getDirName()
      * Return the Resource Name for templates in this plugin, or if specified, the full resource locator
      * for a specific template.
      *
+     * When a template is specified, returns a Laravel view namespace (e.g., "funding::listFunders").
+     * Laravel's view system resolves the actual file (Blade or Smarty) automatically.
+     *
+     * When no template is specified, returns the Smarty resource name for backward compatibility
+     * with Smarty resource registration.
+     *
      * @param string $template path/filename, if desired
      * @param bool $inCore True if a "core" template should be used.
      *
      * @return string
      */
     public function getTemplateResource($template = null, $inCore = false)
     {
-        if ($template) {
-            $bladeTemplatePath = $this->resolveBladeViewPath($template);
+        // When template is specified, return Laravel namespace format
+        // This works with PKPTemplateManager::fetch() which routes through Laravel's view system
+        if ($template !== null) {
+            // Remove .tpl extension and convert slashes to dots
+            $viewPath = preg_replace('/\.tpl$/', '', $template);
+            $viewPath = str_replace(['/', '\\'], '.', $viewPath);
 
-            if (view()->exists($bladeTemplatePath)) {
-                // share the template variables to the blade view
-                $templateManager = TemplateManager::getManager(Application::get()->getRequest());
-                $templateManager->shareTemplateVariables($templateManager->getTemplateVars());
-                
-                return $bladeTemplatePath;
-            }
+            return $this->getTemplateViewNamespace() . '::' . $viewPath;
```

Comment:
this assumes that a corresponding blade view is always available but that may not be case for plugin not migrating to blade yet .